### ISSUE TITLE
Dialog is not opened as modal on Android

### ISSUE THREAD

**Issue by pkfms** [tamagui/tamagui#3061]
### Current Behavior

Dialog/AlertDialog is opened inline on Android

### Expected Behavior

Dialog/AlertDialog should open as modal.
This was working on tamagui 1.115.5

### Tamagui Version

```markdown
1.116.14
```

### Platform (Web, iOS, Android)

```markdown
Android
```

### Reproduction

```markdown
https://github.com/pkfms/starter-free

Dialog/AlertDialog example is copied from docs into the home screen. Not using native prop.
Click "show alert" on android. It is opened inline and not as modal.
```

**Comment by XChikuX** [comment]
Facing the same issue! Thanks for posting this.

**Comment by pkfms** [comment]
I checked the changes between 1.115.5 and the latest version. The problem is that PortalItem is no longer used on native. This was moved to AdaptParent. So when not using adapt, Dialog needs to use PortalItem for it to work on native

**Comment by lukway-dev** [comment]
I have the same problem, 1 month and a half since this error was published and we still have no solution.

**Comment by likeSo** [comment]
Same problem here, now the `AlertDialog` opened inline.

**Comment by natew** [comment]
The problem is in part because React Native doesn't support portals at all. We found a hacky way to get it to work, but it causes issues in other ways. And we can force it to portal though just copying the entire VDOM, but that causes context issues.

**Comment by pmontes-designli** [comment]
and are you working on any fix? pleeeaseeeee dont want to move the modals at the root

**Comment by natew** [comment]
In the latest release I've made it so you can enable the old portal functionality on Android by setting `TAMAGUI_USE_NATIVE_PORTAL=1`. Let me know if that fixes for you.

**Comment by pkfms** [comment]
> In the latest release I've made it so you can enable the old portal functionality on Android by setting `TAMAGUI_USE_NATIVE_PORTAL=1`. Let me know if that fixes for you.

No, that made no difference. I think because when not using adapt, portal is never used, and modal prop has no effect.
https://github.com/tamagui/tamagui/blob/422d0c15d785e9a492c7097b3cf3bd4a51440eca/code/ui/dialog/src/Dialog.tsx#L535

Also, it's the same as this issue https://github.com/tamagui/tamagui/issues/3124#issue-2781262595

**Comment by pkfms** [comment]
I was playing around a bit and managed to get Dialog/AlertDialog working on native with these changes. As is suspected the problem is that PortalItem is never used when not using adapt.
https://github.com/tamagui/tamagui/commit/e198c8f85a954c4d1dc30a988dfc5774baf2f5be
[@tamagui+dialog+1.125.12.patch](https://github.com/user-attachments/files/18913098/%40tamagui%2Bdialog%2B1.125.12.patch)

This is the commit that changed this behavior in 1.116.0. Dialog.tsx DialogPortal and DialogPortalItem. These are the changes I was trying to restore
https://github.com/tamagui/tamagui/commit/0ca50d73dbb2386636580be499c76046def492eb

**Comment by natew** [comment]
Fix going out now
